docker/install-full-database.sh

```diff
@@ -0,0 +1,13 @@
+echo "🥫 Downloading the full MongoDB dump…"
+wget https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.tar.gz
+echo "🥫 Copying the dump to Docker…"
+docker cp openfoodfacts-mongodbdump.tar.gz docker_mongodb_1:/data/db
+echo "🥫 Opening a shell within the Mongo docker…"
+docker exec -it docker_mongodb_1 bash
```

Comment:
How does it work? I don't quite understand how the following script is run inside the container.

Reply:
install-full-database.sh is meant to be run outside
this instruction will be run to open a shell inside
https://www.oreilly.com/library/view/hands-on-devops-with/9781789138054/a9a81868-1652-4e57-a064-0c64de55eefc.xhtml

Reply:
This will be automated as part of the `restructure-docker` branch in the Makefile. Note to myself: `wget` doesn't work on Powershell.